tests/unit_tests/test_league_rater.py

```diff
@@ -0,0 +1,282 @@
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score - config.SCORE_GAIN
+
+
+def test_new_score_defeat_boost(example_league):
+    current_score = LeagueScore(division_id=2, score=5, game_count=30)
+    player_rating = (60.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.DEFEAT,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score - config.SCORE_GAIN - config.NEGATIVE_BOOST
+
+
+def test_new_score_victory_highest_division_no_boost(example_league):
+    current_score = LeagueScore(division_id=3, score=5, game_count=30)
+    player_rating = (240.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN
+
+
+def test_new_score_victory_highest_division_boost(example_league):
+    current_score = LeagueScore(division_id=3, score=5, game_count=30)
+    player_rating = (380.0, 0.0)
+
+    new_score = LeagueRater.rate(
+        example_league,
+        current_score,
+        GameOutcome.VICTORY,
+        player_rating,
+    )
+
+    assert new_score.division_id == current_score.division_id
+    assert new_score.game_count == current_score.game_count + 1
+    assert new_score.score == current_score.score + config.SCORE_GAIN + config.HIGHEST_DIVISION_BOOST
+
+
+def test_placement(example_league, unplaced_player_score):
+    rating = 150 - config.RATING_MODIFIER_FOR_PLACEMENT
+
+    new_score = LeagueRater._do_placement(example_league, unplaced_player_score, rating)
+
+    assert new_score.division_id == example_league.divisions[1].id
+    assert new_score.score == 5
+
+
+def test_placement_high_rating(example_league, unplaced_player_score):
+    rating = 1500 - config.RATING_MODIFIER_FOR_PLACEMENT
+
+    new_score = LeagueRater._do_placement(example_league, unplaced_player_score, rating)
+
+    assert new_score.division_id == example_league.divisions[-1].id
+    assert new_score.score == 10
+
+
+def test_placement_low_rating(example_league, unplaced_player_score):
+    rating = -500 - config.RATING_MODIFIER_FOR_PLACEMENT
+
+    new_score = LeagueRater._do_placement(example_league, unplaced_player_score, rating)
+
+    assert new_score.division_id == example_league.divisions[0].id
+    assert new_score.score == 0
+
+
+def test_new_player(example_league):
+    current_score = LeagueScore(division_id=None, score=None, game_count=None)
```

Comment:
I think we should adopt the convention to set the `game_count` to zero for new players, unless you see any problems with that.